app/views/example.html

```diff
@@ -143,6 +143,9 @@
     {% block headerEnd %}
     {% endblock %}
 
+    {% block pageBreadcrumbs %}
```

Comment:
Would it be good to rename the block as `breadcrumbs` as it is not within the `main` anymore? I understand that this involves updating all HTML files which are currently referencing `pageBreadcrumbs` block. Maybe worth updating it?